src/components/Preview.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Preview from './Preview';
+import { cleanup } from '@testing-library/react';
+
+afterEach(cleanup);
+
+describe('<Preview/>', () => {
+  it('renders without crashing', () => {
+    global.renderWithRedux(<Preview />);
+  });
+
+  it('renders appropriate content', () => {
+    // render
+    const { getByTestId } = global.renderWithRedux(
+      <Preview
+        expanded={true}
```

Comment:
FYI, for boolean props specifically, if you are passing in `true` you can also omit the value. In other words, you could simplify this to 

```suggestion
        expanded
```

Reply:
Done